componentified-todo-list/banafshe/to-do-list/src/components/addTodo.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+
+
+const AddTodo = ({addTopHandler, addBottomHandler}) => {
+    const [todoTitle, setTodoTitle] = useState("");
+    const handleInputChange = (event) => {
+        setTodoTitle(event.target.value);
+    };
+    const idGenerator = () =>{
+        return Math.abs(Math.random()*10000) ;
```

Comment:
__[MEDIUM]__  
There is a probability that this function does not generate unique ids.